source/codegen/metadata/nixnet/enums_addon.py

```diff
@@ -8,5 +8,6 @@
     "PropertyValue": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"],
     "DBPropertyValue": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"],
     "StateValue": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"],
-    "SubPropertyValue": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"]
+    "SubPropertyValue": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"],
+    "FrameFlags": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"]
```

Comment:
Note: This was failing the build after adding `force-include` enums like FrameFlags to the enum validation.